Packs/TrustwaveFusion/Integrations/TrustwaveFusion/README.md

```diff
@@ -0,0 +1,5 @@
+This README contains the full documentation for your integration.
+
+You auto-generate this README file from your integration YML file using the `demisto-sdk generate-docs` command.
+
+For more information see the [integration documentation](https://xsoar.pan.dev/docs/integrations/integration-docs).
```

Comment:
please generate a readme to the integration you can use the command generate-docs and use the command examples file to provide examples of the commands in the integration.

Reply:
See https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md for more information how to use `demisto-sdk generate-docs` command.